scripts/zones/Port_Bastok/Zone.lua

```diff
@@ -13,12 +13,13 @@ local zoneObject = {}
 zoneObject.onInitialize = function(zone)
     zone:registerTriggerArea(1, -112, -3, -17, -96, 3, -3)     -- event COP
     zone:registerTriggerArea(2, 53.5, 5, -165.3, 66.5, 6, -72) -- drawbridge area
-    xi.conquest.toggleRegionalNPCs(zone)
+    xi.conquest.setTravelingMerchants(zone, 1, ID.npc.TRAVELING_MERCHANTS)
     xi.events.starlightCelebration.applyStarlightDecorations(zone:getID())
 end
 
-zoneObject.onConquestUpdate = function(zone, updatetype)
-    xi.conq.onConquestUpdate(zone, updatetype)
+zoneObject.onConquestUpdate = function(zone, updateType)
+    xi.conq.onConquestUpdate(zone, updateType)
+    xi.conquest.setTravelingMerchants(zone, updateType, ID.npc.TRAVELING_MERCHANTS)
```

Comment:
Try changing these to xi.conq.setTravelingMerchants?  Every call to zoneObject.onConquestUpdate uses that nomenclature instead of the full xi.conquest.  I don't see a reason for it, but there may be some collision/reference I'm not seeing.

If that doesn't work, You could remove all of these from their specific Zone files and wrap the original location inside of xi.conquest.onConquestUpdate with a zoneID check for only the 3 listed here.

Reply:
Seems to me the `xi.conq` one is wrong because the global has `xi.conquest = xi.conquest or {}` but I can't get evidence that either of them are being called. None of my prints do anything. Neither does PrintToPlayer, grabbing the reference from zone. Totally lost